README.rst

```diff
@@ -114,9 +125,6 @@ Documentation
 | Stable  | |stable|    | `Documentation of the stable (i.e. most recent release) branch. <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/>`_ |
 +---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
 
-Tutorials
----------
```

Comment:
Lets not remove the tutorial section, it makes it harder to find in a wall of text in the intro that most wont read fully.